app/containers/Leaderboard/Leaderboard.js

```diff
@@ -20,11 +20,69 @@ export const Leaderboard = React.createClass({
 		this.props.dispatch(getLeaderboard());
 	},
 
-	render() {
+    getInitialState() {
+		return {
+			leaderSort: 'byStateCount',
+		};
+	},
+
+    leadersByStateCount() {
+        const leaders = this.props.leaderboardData.leaders || [];
+        const flatLeaders = flattenLeaders(leaders).sort(function (a,b) {
+            return cmp(a.statesCount, b.statesCount) || cmp(a.score, b.score);
+        }).reverse();
+        console.log("called");
```

Comment:
remove console.log calls, they might be confusing for later development. (also unnecessary for users to see)